test/com/google/javascript/jscomp/GoldenFileComparer.java

```diff
@@ -104,6 +105,33 @@ private static void compileAndCompare(
     compare(compiledSource, referenceSource);
   }
 
+  /**
+   * Compile sourceFileName with given CompilerOptions and check that all lines of code from
+   * goldenFileName are present in compiled sourceFile.
```

Comment:
Also mention that the golden file lines must appear in the same order as in the golden file, not just in any arbitrary order.

Reply:
Updated.